viewer/packages/sector-parser/src/types.ts

```diff
@@ -95,13 +95,8 @@ export type GeometryProcessingPayload = {
   data: ArrayBuffer;
 };
 
-export type TypedArrayConstructor =
-  | Int8ArrayConstructor
-  | Uint8ArrayConstructor
-  | Int16ArrayConstructor
-  | Uint16ArrayConstructor
-  | Int32ArrayConstructor
-  | Uint32ArrayConstructor
-  | Uint8ClampedArrayConstructor
-  | Float32ArrayConstructor
-  | Float64ArrayConstructor;
+export type ParsedGeometry = {
+  type: RevealGeometryCollectionType;
+  geometryBuffer: THREE.BufferGeometry;
+  instanceId?: string;
```

Comment:
Why is this `string` somewhere and `number` elsewhere?

Reply:
Should be string consequently everywhere now